src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/ProducerQueryBuilder.java

```diff
@@ -18,19 +17,18 @@ public class ProducerQueryBuilder {
   @PersistenceContext
   private EntityManager entityManager;
 
-  public List<Producer> findAllProducersByProducerName(Integer limit, Integer offset,
+  public List<String> findAllProducersByProducerName(Integer limit, Integer offset,
```

Comment:
It was better before

Reply:
What do you mean? Now probing-operations just returns a list of string because the object required by the fe is created by probing-api

Reply:
I mean you need to work with Entities, this is not a the right use of an ORM in my opinion. You should retrieve producers Entity and after return to FE only the name, but he service is not created on FE purposes.
But if you want use this approach please go forward, but my question is: Why are you using ORM?

Reply:
I ask you to agree on the implementation, just to clarify the points:
- The repository should return a list of Producer, `List<Producer>`, entities that have the `producerName` attribute.
- In the service implementation, should I return the `SearchProducerNameResponse` object, which has a list of strings as an attribute, `List<String> content`, and do the conversion on the _probing-eservice-operations_ side? Otherwise, if I change the string list in SearchProducerNameResponse to a list of Producer objects, I should also update the implementation in _probing-api_.